tools/free_energy/alchemical_analysis/alchemical_analysis.xml

```diff
@@ -0,0 +1,85 @@
+<tool id="alchemical_analysis" name="Alchemical Analysis" version="1.0.2">
+    <description>Analysis of alchemical free energy calculations</description> 
+    <requirements>
+         <requirement type="package" version="1.0.2">alchemical-analysis</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+       ln -s '$datain' ./data.tar &&
+       tar xf ./data.tar && 
+       mkdir ./RESULTS/ &&
+            #if $skiplambda = "":
```

Comment:
```suggestion
            #if $skiplambda == "":
```

Reply:
@tsenapathi, there is also a test file missing

Reply:
Fixed it. Thanks @simonbray !!